presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -414,11 +306,238 @@ as a part of SQL query by appending name with the table.
     ---------+------------------------------------------------------------------------------+-------------+--------------+--------------------+-----------------------------+--------------------------+----------------------+------------------+-------------------------------------------+--------------------------------------------+--------------+---------------+-------------
        0     | s3://my-bucket/ctas_nation/data/9f889274-6f74-4d28-8164-275eef99f660.parquet | PARQUET     |           25 |               1648 | {1=52, 2=222, 3=105, 4=757} | {1=25, 2=25, 3=25, 4=25} | {1=0, 2=0, 3=0, 4=0} |  NULL            | {1=0, 2=ALGERIA, 3=0, 4= haggle. careful} | {1=24, 2=VIETNAM, 3=4, 4=y final packaget} | NULL         | NULL          | NULL
 
+SQL Support
+-----------
+
+The Iceberg connector supports querying and manipulating Iceberg tables and schemas
+(databases). Here are the examples of SQL Operations supported from Presto :
```

Comment:
Here are the examples of SQL Operations supported from Presto :
-->
Here are some examples of the SQL operations supported by Presto :